Makefile

```diff
@@ -102,6 +102,8 @@ ARFLAGS		=	rcs
 ELFFLAGS	+=	-static
 ELFFLAGS	+=	$(EXTRA_ELFFLAGS)
 
+LDFLAGS	+= -L$(build_dir) -lrpmi
```

Comment:
Add these to `ELFFLAGS` just after `-static`

Reply:
Also, the top-level Makefile change should be a separate patch.

Reply:
Done